catalog_tools/utils/binning.py

```diff
@@ -0,0 +1,73 @@
+import numpy as np
+import decimal
+
+
+def normal_round_to_int(x: float) -> int:
+    """
+    Rounds a float number x to the closest integer.
+
+    Args:
+        x: decimal number that needs to be rounded
+
+    Returns:
+        Rounded value of the given number.
+    """
+
+    sign = 1 if (x >= 0) else -1
+    x = abs(x)
+
+    if x - np.floor(x) < 0.5:
+        return int(sign * np.floor(x))
+    return int(sign * np.ceil(x))
+
+
+def normal_round(x: float, n: int) -> float:
```

Comment:
if n is optional, shouldn't it be set to 0 as a default?

Reply:
of course, thanks